### ISSUE TITLE
Issues concerning data

### ISSUE THREAD

**Issue by ppanzx** [vinid/path_eval#6]
Thank you for your nice work in pathological pre-training!
1. I'm wondering if there are more specific annotations available for the OpenPath test set. Additionally, could you please upload the raw data of the OpenPath test set to Google Cloud?
2. I'm currently in the process of downloading the PathPedia and DigestPath data to evaluate the performance of PLIP. However, I'm encountering difficulties when trying to download the data from their official website. (e.g., not being able to sign up, slow download, etc.) Could you upload the data using Google Drive or other cloud?
Thank you!

**Comment by vinid** [comment]
Hello!

Thank you so much for reaching out! We appreciate your kind words!

Concerning Twitter data, our ability to share it is limited by Twitter's privacy and control policies. We can only distribute Tweet IDs and we recommend the usage of the Twitter API for reconstruction.

Regarding PathPedia and DigestPath we are not able to share the raw data. These databases operate under specific terms of use that forbid unauthorized redistribution, due to privacy and data protection considerations. 

Let us know if we can be of help in any other way!

**Comment by ppanzx** [comment]
The open-source checkpoint available in [hugging face](https://huggingface.co/vinid/plip/tree/main) does not match the version in your  [source code](https://github.com/PathologyFoundation/plip/blob/main/training_model/clip.py#L91). Could you kindly upload a checkpoint that corresponds to the OpenAI version?

**Comment by vinid** [comment]
We are working on a new version of the entire source code: https://github.com/PathologyFoundation/plip/tree/dev

hopefully, we should have all that is needed soon

**Comment by ppanzx** [comment]
Thank you for your trouble!

**Comment by Sharmalm** [comment]
Hello @vinid , Thankyou for this work on pathological pre-training.
I am an undergraduate student and doing a research project in my university. Can you please share twitter data with me, twitter api now is very expensive and unfortunately i can't afford it. Data will only be use for educational pruposes. 

Thanks and regards :)

**Comment by vinid** [comment]
Unfortunately we cannot share the full data due to how it has been collected. 

The recent changes in Twitter/X APIs pricing and usage as also affected our own data collection pipeline: our collection was an ongoing effort that we had to put on pause due to the cost of the API access.

**Comment by Sharmalm** [comment]
Thank you for your response.
